quickstart/known-limitations.md

```diff
@@ -10,7 +10,7 @@ Mayastor has no snapshot or cloning capabilities.
 
 ## Volumes are "Highly Durable" but without multipathing are not "Highly Available"
 
-Mayastor Volumes can be configured \(or subsequently re-configured\) to be composed of 2 or more "children" or "replicas"; causing synchronously mirrored copies of the volumes's data to be maintained on more than one worker node and Disk Pool. This contributes additional "durability" at the persistence layer, ensuring that viable copies of a volume's data remain even if a Disk Pool device is lost.
+Mayastor Volumes can be configured \(or subsequently re-configured\) to be composed of 2 or more "children" or "replicas"; causing synchronously mirrored copies of the volumes's data to be maintained on more than one worker node and DiskPool. This contributes additional "durability" at the persistence layer, ensuring that viable copies of a volume's data remain even if a DiskPool device is lost.
 
-However, a Mayastor volume is currently accessible to an application only via a single target instance \(NVMe-oF\) of a single Mayastor pod. If that Mayastor pod ceases to run \(through the loss of the worker node on which it's scheduled, execution failure, crashloopbackoff etc.\) then there will be no viable I/O path to any remaining healthy replicas and access to data on the volume cannot be maintained.
+A Mayastor volume is currently accessible to an application only via a single target instance \(NVMe-oF\) of a single Mayastor pod. However, if that Mayastor pod ceases to run \(through the loss of the worker node on which it's scheduled, execution failure, crashloopbackoff etc.\) the switch-over feature quickly detects the failure and spawns a new nexus to ensure I/O continuity.
```

Comment:
Since we have not used the word nexus in this context, I'd propose something like "the HA switch-over module detects the failure and moves the target to a healthy worker node to ensure I/O continuity"?